maxwellgithinji/Stack-Overflow-lite
# Chage fetch url 161052237

**What is this PR supposed to do?**
change the local app url to that of heroku

**How can this be manually tested**
* Fire up the API server
* Run the node server
* ensure the app is running on heroku server

 **Any background context you want to provide**
the UI is hosted on gh-pages with the nameUX

**What are the relevant PT stories?**
https://www.pivotaltracker.com/story/show/161052237

**Questions**
